### PR TITLE
Take coauthors into account

### PR DESCRIPTION
**Pull request by varkor**
Fixes https://github.com/rust-lang/thanks/issues/16.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -179,6 +180,29 @@ fn get_versions(repo: &Repository) -> Result<Vec<VersionTag>, Box<dyn std::error
...
+    if let Some(msg) = commit.message_raw() {
+        lazy_static::lazy_static! {
+            static ref RE: Regex =
+                RegexBuilder::new(r"^Co-authored-by: (?P<name>.*) <(?P<email>.*)>")
+                    .case_insensitive(true)
+                    .build()
+                    .unwrap();
+        }
+
+        for line in msg.split('\n').rev() {
```
Comment (Mark-Simulacrum) [inline src/main.rs]: Why not `lines().rev()`?

File: src/main.rs [hunk 2]
```diff
@@ -197,9 +221,13 @@ fn build_author_map(
     for oid in walker {
         let oid = oid?;
         let commit = repo.find_commit(oid)?;
-        let commit_author = Author::new(commit.author());
-        let author = mailmap.canonicalize(&commit_author);
-        author_map.add(author, oid);
+        // let mut commit_authors = commit_coauthors(&commit);
```
Comment (Mark-Simulacrum) [inline src/main.rs]: Looks like this needs to be uncommented?

File: src/main.rs (hunk 1 above)
Comment (varkor) [reply src/main.rs]: Because I didn't spot it 😄Fixed.

File: src/main.rs (hunk 2 above)
Comment (varkor) [reply src/main.rs]: Whoops, fixed.

**Pull request merged**